.github/workflows/ci.yml

```
@@ -19,6 +19,20 @@ jobs:
+      name: Install pnpm
+      with:
+        version: 10
+        run_install: false
+        working-directory: src/ui
+  
+    - name: Install Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: 20
+        cache: 'pnpm'
+        working-directory: src/ui
```

actionlint (1.7.7)

input "working-directory" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)